eos-updater/tests/eos_updater.py

```diff
@@ -0,0 +1,428 @@
+#!/usr/bin/python3
+# -*- coding: utf-8 -*-
+#
+# Copyright © 2019 Endless Mobile, Inc.
```

Comment:
```suggestion
# Copyright © 2019 Endless OS Foundation LLC
```

Reply:
At the time, we weren’t yet Endless OS Foundation (that happened in 2020), which is why I kept this as Endless Mobile. I believe that’s correct, although it’s all a bit moot because the copyright ownership has transferred.